--- unbound.conf    2025-04-03 21:56:21.409697372 +0200
+++ unbound.conf    2025-04-03 21:56:37.642453621 +0200
@@ -68,9 +68,9 @@
         name: "vpn."
         forward-addr: 127.0.0.11
         forward-no-cache: yes
-    # forward-zone:
-    #     name: "."
-    #     forward-tls-upstream: yes
-    #     # Cloudflare
-    #     forward-addr: 1.1.1.1@853#cloudflare-dns.com
-    #     forward-addr: 1.0.0.1@853#cloudflare-dns.com
+    forward-zone:
+        name: "."
+        forward-tls-upstream: yes
+        # Cloudflare
+        forward-addr: 1.1.1.1@853#cloudflare-dns.com
+        forward-addr: 1.0.0.1@853#cloudflare-dns.com
